src/Entities/CompleteApplicantEntity.php

```diff
@@ -159,9 +159,7 @@ public function getAssessmentDay(): ?string
         return $this->assessmentDay;
     }
 
-    /**
```

Comment:
Why was this deleted? This getter is now inconsistent with the others in this file.
Also there are now two blank lines between getters - only one is needed.